Makefile

```diff
@@ -389,17 +389,17 @@ generate-go: $(CONTROLLER_GEN) $(MOCKGEN) $(CONVERSION_GEN) $(KUBEBUILDER) $(KUS
 		--go-header-file=./hack/boilerplate/boilerplate.generatego.txt
 
 .PHONY: generate-manifests
-generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
-	cd $(APIS_DIR) && ../$(CONTROLLER_GEN) \
-		paths=./... \
-		crd:crdVersions=v1 \
-		output:crd:dir=../$(CRD_ROOT) \
-		output:webhook:dir=../$(WEBHOOK_ROOT) \
-		webhook
+generate-manifests: ## Generate manifests e.g. CRD, RBAC etc.
```

Comment:
I think `$(CONTROLLER_GEN)` should still be on this line. It makes sure that it is built before executing everything below.

Reply:
Ah right! in case someone calls this phony directly then it would break, ofcourse make generate works since that calls generate-go which builds all the binaries before moving forward to generate manifests.